CODEOWNERS

```diff
@@ -0,0 +1,65 @@
+# https://help.github.com/en/articles/about-code-owners
+# Documentation: https://github.com/hmcts/sds-flux-config/blob/master/CODEOWNERS.md
+
+CODEOWNERS @hmcts/platform-operations
+
+# Restricting access to platops will we cleanup
```

Comment:
what's this comment mean 'will we cleanup'

Reply:
Don't know tbh, used cnp-flux-config as a template and it has this

Reply:
prob just remove